bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/i18n/gardena.properties

```diff
@@ -38,10 +38,8 @@ thing-type.config.gardena.account.apiKey.label = API Key
 thing-type.config.gardena.account.apiKey.description = The Gardena smart system integration API key
 thing-type.config.gardena.account.connectionTimeout.label = Connection Timeout
 thing-type.config.gardena.account.connectionTimeout.description = The timeout in seconds for connections to Gardena smart system integration API
-thing-type.config.gardena.account.email.label = Email
-thing-type.config.gardena.account.email.description = Email address for logging in to Gardena smart system
-thing-type.config.gardena.account.password.label = Password
-thing-type.config.gardena.account.password.description = Password for logging in to Gardena smart system
+thing-type.config.gardena.account.apiSecret.label = API Secret
+thing-type.config.gardena.account.apiSecret.description = The Gardena smart system integration API key
```

Comment:
It seems this file needs to be regenerated as in bridge definition string is different: "The Gardena smart system integration API **secret**"